bundles/admin/admin-layereditor/components/Button.jsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Button from 'antd/lib/button';
+import AntButton from 'antd/lib/button';
 import 'antd/lib/button/style/css';
 
-export const ButtonComponent = (props) => {
-    const {text, ...other} = props;
-    return (
-        <Button {...other}>{text}</Button>
-    );
-};
+export const Button = (props) => (
+    <AntButton {...props}>{props.children}</AntButton>
```

Comment:
It would be cleaner no to pass `children` twice:
```
export const Button = ({children, ...other}) => (
<AntButton {...other}>{props.children}</AntButton>
);
```